### PR TITLE
Added feature for helping user to bring required things according to weather.

### PR DESCRIPTION
**Pull request by 0904-mansi**

### CODE REVIEW

**Review by trinhcaokhoa — COMMENTED** [review]
Clean code checked.

**Comment by grace-omotoso** [comment]
@0904-mansi please, fix merge conflicts when you can. I know it is pretty too early for you now.

**Comment by 0904-mansi** [comment]
> @0904-mansi please, fix merge conflicts when you can. I know it is pretty too early for you now.

Sorry for the inconvenience. I removed the conflicts.

File: src/Components/Js/requiredThings.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import Sanitation from '../assets/requiredThings/Sanitation.png';
+import MosquitoRepellent from '../assets/requiredThings/MosquitoRepellent.png';
+import Cap from '../assets/requiredThings/Cap.png';
+import SandScarf from '../assets/requiredThings/Scarf.png';
+import Wipes from '../assets/requiredThings/Wipes.jpg';
+
+
+
+
+const requiredThings = {
```
Comment (grace-omotoso) [inline src/Components/Js/requiredThings.js:21]: What do we bring when it is sunny? Sunscreen, Cap, Sunglass, etc. Could you also include sunny weather?

Comment (0904-mansi) [reply src/Components/Js/requiredThings.js:21]: sure mam

**Review by grace-omotoso — COMMENTED** [review]
LGTM. @Bhawna1203, @norachams can we have one of you test the change before it gets merged?

**Comment by norachams** [comment]
> LGTM. @Bhawna1203, @norachams can we have one of you test the change before it gets merged?

Just tested it. It looks good!

**Pull request merged**